app/views/courses/financial_support/_bursary.html.erb

```diff
@@ -16,11 +16,10 @@
   </p>
 
   <p class="govuk-body">
-    You may also be eligible for a <%= link_to "loan while you study","https://getintoteaching.education.gov.uk/funding-my-teacher-training/tuition-fee-and-maintenance-loans", class:"govuk-link" %>.
+    You may also be eligible for a <%= link_to "loan while you study" , "https://getintoteaching.education.gov.uk/funding-my-teacher-training/tuition-fee-and-maintenance-loans", class:"govuk-link" %> - note that you'll have to apply for <%= link_to "undergraduate student finance" , "https://www.gov.uk/student-finance", class:"govuk-link" %>.
```

Comment:
Can we make the spacing consistent here? `" , "` -> `", "` and `class:"govuk-link"` -> `class: "govuk-link"`? Same in the other two.